### PR TITLE
Fix CHANGELOG

### PR DESCRIPTION
**Pull request by botanical**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,28 +26,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
...
   - Deploy ORCA with Cumulus, see `example/cumulus-tf/orca.tf` and `example/cumulus-tf/terraform.tfvars.example`
   - Add `CopyToGlacier` step to [example IngestAndPublishGranule workflow](https://github.com/nasa/cumulus/blob/master/example/cumulus-tf/ingest_and_publish_granule_workflow.asl.json)
 
 ### Changed
 
-- `<prefix>-lambda-api-gateway` IAM role used by API Gateway Lambda now supports accessing all buckets defined in your `buckets` variable except "internal" buckets
 - **CUMULUS-2362**
   - Logs endpoints only work with Metrics set up
-
+- **CUMULUS-2376**
```
Comment (markdboyd) [inline CHANGELOG.md:37]: These lines should also be moved: https://github.com/nasa/cumulus/blame/master/CHANGELOG.md#L63-L64

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+
+### Added
+- `tf-modules/cumulus` module now supports a `cmr_custom_host` variable that can be used to set to an arbitrary host for making CMR requests (e.g. `https://custom-cmr-host.com`).
+- Added `buckets` variable to `tf-modules/archive`
+### Changed
```
Comment (markdboyd) [inline CHANGELOG.md:12]: Nit: please include newlines between the headers

Comment (botanical) [reply CHANGELOG.md:14]: [`a988867` (#2121)](https://github.com/nasa/cumulus/pull/2121/commits/a988867e4274bc68de14a1c2cc265f3f79bdeac6)

**Review by markdboyd — APPROVED** [review]
Thanks!

**Pull request merged**